CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -167,18 +167,28 @@ public static function checkDuplicate($params, &$duplicates) {
    * Get the payment processor (array) for a recurring processor.
    *
    * @param int $id
-   * @param string $mode
-   *   - Test or NULL - all other variants are ignored.
    *
    * @return array|null
    */
-  public static function getPaymentProcessor($id, $mode = NULL) {
+  public static function getPaymentProcessor($id) {
```

Comment:
this is safe - function not called from anywhere prior to this PR (we kept it when it stopped being called as a 'better' choice